.changes/unreleased/Under the Hood-20241023-180425.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Under the Hood
+body: Added validation warnings for invalid time spines in where filters of saved queries.
```

Comment:
I would amend this slightly (and the PR title too) - this is validating `granularities`, not `time spines`. The granularity they put in might reference a standard granularity (not associated with a time spine) or a custom granularity (which is associated with a time spine, though the time spine is not what's being validated here).